### ISSUE TITLE
is firmware.bin in the root directory the correct file?

### ISSUE THREAD

**Issue by spunkyblob** [The-300lb-Gorilla/Ender-2-Pro-BTT-V3#1]
Hi - thanks for your work on this - I am still a little confused as I have downloaded the firmware.bin and flashed my BTT V3 but it still says 'ender 3' and the menu system is exactly the same as when I flashed it with the original os. Cheers

**Comment by The-300lb-Gorilla** [comment]
I threw that together from the Ender 3 base configuration, I might have missed the machine name change, does the Ender 2 Pro function correctly?
Other than the name, all other functions ok?

**Comment by The-300lb-Gorilla** [comment]
I am working on a follow up version for the version 3 board. Would you like me to message you once it is complete?

**Comment by spunkyblob** [comment]
It seems to work fine. Struggling to dial in my printer at the moment due to the lack of z offset in the menu system. Is this something that can be added?

**Comment by The-300lb-Gorilla** [comment]
Okay. Let me run through the menu system and make sure it is in fact there. Once I do this, I will get back to you as soon as I am finished. Just a side note, I did design this conversion to be with a Main Board and a TFT35 Touch Screen. Thankfully I have a stock screen module, so I can hook it up and run through the menus like you can. Using the Marlin Mode on the TFT35 is the same as a stock display, but that is a story for another time. I will get right on this. Stand by. 
NOTE: This would go a lot faster if I had an email address to conduct trouble shooting through. you would not have to keep checking here to see if there is a response.

**Comment by The-300lb-Gorilla** [comment]
One other thing.
Did you do a manual bed traming?  (This is leveling by the 4 corners, Remember to lower you bed as far as possible before doing this, the 3 axis will home and then go to the first corner, you can keep pushing the knob and it will keep going from corner to corner until you have the corners set using a piece of paper, you can go around the bed on the corners as many times as you want, when you are satisfied with the results, turn the knob from Next to back to left to DONE)
Go back out of that menu, and do a Bed Leveling (This is the Manual Mesh Bed Leveling), at each of the 9 points on the bed use a piece of paper and place it under the nozzle and turn the knob to raise or lower the nozzle until there is resistance drag on the paper, when you are satisfied with the result at each point, push the knob and it will move to the next point. Do all 9 points, and then it will say bed leveling done. Remember to Store Settings in the Bed Leveling Menu.
[Setting the Z Height Offset.pdf](https://github.com/The-300lb-Gorilla/Ender-2-Pro-BTT-V3/files/8936314/Setting.the.Z.Height.Offset.pdf)

on the screen. You can use the "Bed Z" in the Motion Menu to set your offset of the nozzle to the bed. Option 2: you can go to the Edit Mesh option in the Bed Leveling menu and select the Edit Mesh, then select Z Value, and adjust the offset from the nozzle to bed that way. NOTE: you have to do a Mesh Bed Leveling before you use option 2, and always remember to "Store Settings"
back in the Bed Leveling Menu.

**Comment by The-300lb-Gorilla** [comment]
There is now an instructional PDF on setting the Z height offset on the Ender-2-Pro-BTT-V3 branch.
Still checking over the firmware code.

**Comment by The-300lb-Gorilla** [comment]
Manual Mesh Bed Leveling is functioning as set by the firmware version.
Read the PDF on Z Height offset setting after mesh bed leveling

**Comment by The-300lb-Gorilla** [comment]
Okay. I just dropped a recompiled firmware.bin file here:
https://github.com/The-300lb-Gorilla/Ender-2-Pro-BTT-V3/tree/main/Pre%20Compiled%20Firmware%20Version%203
I ordered a BTT-SKR-Mini E3 V3 board for my other Ender 2 Pro, will be here tomorrow. I will be using a stock LCD screen, same as you. So, if the firmware I just dropped gives you issues, I can trouble shoot on my end with the same setup.
Since I only wrote firmware for those 2 boards, I figured I might as well be using both myself. Hopefully the firmware I dropped fixes the issue.
